esm/mixin/inner-html.js

```diff
@@ -0,0 +1,19 @@
+import {parseFromString} from '../shared/parse-from-string.js';
+import {
+  ignoreCase,
+} from '../shared/utils.js';
+import {
+  CUSTOM_ELEMENTS
+} from '../shared/symbols.js';
+
```

Comment:
for consistency sake, please import constants on top and don't use multiple lines if the import fits into 80 chars/column.

Reply:
Done.